docker/docker-compose.yml

```diff
@@ -0,0 +1,49 @@
+version: "3"
+services:
+  bitcoin-core:
+    container_name: bitcoin-core
+    image: ruimarinho/bitcoin-core:0.19.1-alpine
+    command:
+      -printtoconsole
+      -testnet=1
+      -txindex=1
+      -server=1
+    environment:
+      - BITCOIN_DATA=/data
+    volumes:
+      - ${DATA_VOL}:/data
+  bitcoin-ion:
+    container_name: bitcoin-ion
+    image: node:10.20.1-alpine3.11
+    command: "npm run bitcoin"
+    working_dir: /ion
+    links:
+      - "bitcoin-core"
+      - "mongo"
+    volumes:
+      - ./ion:/ion
+  ipfs:
+    container_name: ipfs
+    image: node:10.20.1-alpine3.11
+    command: "npm run ipfs"
+    working_dir: /ion
```

Comment:
This is an interesting approach, different to what I was thinking, but certainly easier to compose... I was thinking of having images for ION services. Do you see this as THE way to compose and run? Or this is more of a short term solution?

Reply:
Short term solution, for publishing ION containers, it makes sense to wrap a nodejs container with each one for ION services.  This also means that we would either need to redirect the configuration for these containers to a volume or distributed configuration specific to each service.  The latter seems to be the clean way to promote reuse.

From a scaling perspective, having these all be separate independent containers makes sense.  For instance I can use a shared IPFS instance and not use this container.